code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -208,14 +207,17 @@
 		balloon_alert(user, "anchor!")
 		return ITEM_INTERACT_SUCCESS
 	if(default_deconstruction_screwdriver(user, "thermo-open", "thermo-0", tool))
-		change_pipe_connection(panel_open)
+		update_appearance()
 		return ITEM_INTERACT_SUCCESS
 
 /obj/machinery/atmospherics/components/unary/thermomachine/wrench_act(mob/living/user, obj/item/tool)
+	if(!anchored)
+		balloon_alert(user, "anchor!")
```

Comment:
```suggestion
		balloon_alert(user, "anchored")
```

`!` in balloon alerts should only be used in "failure" messages, see https://hackmd.io/@tgstation/SJIle5wYd for more info

Reply:
It returns when player failed to rotate the thermomachine if it's not anchored